frontend/src/components/pages/CardDetails/ProjectDetails.jsx

```diff
@@ -38,10 +44,12 @@ function LinkToYourWork({
   const classes = useStyles();
   return (
     <Paper className={classes.sectionPaper} variant="outlined">
-      <Typography variant="subtitle2">Link to your work:</Typography>
+      <Typography variant="subtitle2" className={classes.text}>
+        Link to your work:
+      </Typography>
 
       {currentLinkSubmission ? (
-        <Typography>
+        <Typography className={classes.text}>
```

Comment:
It's strange that the subtitle and normal text are given the same size. Are you sure?